src/nextjournal/clerk/render.cljs

```diff
@@ -765,7 +768,13 @@
         default-loading-view))))
 
 (def render-mathjax mathjax/viewer)
-(def render-code code/viewer)
+
+(defn render-code [value]
+  (let [ref (use-ref nil)]
```

Comment:
@mhuebert is `use-effect` right choice here?  Effect cleanup has the codemirror instance in scope in this way. Do you think there's a better way to do this?